.circleci/config.yml

```diff
@@ -10,6 +10,31 @@ jobs:
     working_directory: /go/src/github.com/banzaicloud/productinfo
     steps:
       - checkout
+       
+      - run:
+          name: Install license checker
+          command: curl https://raw.githubusercontent.com/goph/licensei/master/install.sh | bash
```

Comment:
For now, locking to a specific version would be a good idea (updated in pipeline too): 

`curl https://raw.githubusercontent.com/goph/licensei/master/install.sh | bash -s v0.0.6`